modules/cms/models/ThemeLog.php

```diff
@@ -77,12 +77,6 @@ public static function add(HalcyonModel $template, $type = null)
         $newContent = $template->toCompiled();
         $oldContent = $template->getOriginal('content');
 
-        if ($newContent === $oldContent && $templateName === $oldTemplateName && !$isDelete) {
-            traceLog($newContent, $oldContent);
-            traceLog('Content not dirty for: '. $template->getObjectTypeDirName().'/'.$template->fileName);
-            return;
```

Comment:
This would change the behaviour by removing the short circuit that would occur when no changes are present; thus causing this method to always add an entry even when there are no changes.

Reply:
Didn't see the return statement

Reply:
Only removing the traceLog would be the correct way to go?

Reply:
Yeppers